eng/pipelines/templates/jobs/tests.yml

```diff
@@ -44,27 +51,31 @@ jobs:
           scriptPath: '$(Build.BinariesDirectory)/sdk-tools/scripts/python/verify_agent_os.py'
           arguments: $(OSName)
 
-      - task: DotNetCoreInstaller@0
-        displayName: 'Use .NET Core sdk $(DotNetCoreVersion)'
+      - task: DotNetCoreInstaller@2
```

Comment:
@danieljurek looks like all the .NET Test pipelines are failing after I changed the dotnet variables the other day. This should fix those.